packages/calcite-components/src/components/checkbox/checkbox.tsx

```diff
@@ -189,14 +189,6 @@ export class Checkbox
     }
   };
 
-  clickHandler = (): void => {
```

Comment:
this function is useless since calling `toggle` directly produces the same effect